Gruntfile.js

```diff
@@ -253,10 +259,12 @@ module.exports = function(grunt) {
                     '<%= meta.metaJsFiles %>',
                     'site/src/**/*'
                 ],
-                tasks: ['site']
+                tasks: ['site'],
+                options: {
+                    livereload: true
```

Comment:
Like it! Does this mean we can now do `watch:components site:serve`?

Is it worth specifying the port number here (rather than using the default) as we're reliant on it elsewhere?